src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -127,119 +126,235 @@ private function getPath(string $resourceShortName, array $operation, bool $coll
      *
      * @return \ArrayObject
      */
-    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, ResourceMetadata $resourceMetadata, array $mimeTypes) : \ArrayObject
+    private function getPathOperation(string $operationName, array $operation, string $method, bool $collection, string $resourceClass, ResourceMetadata $resourceMetadata, array $mimeTypes, \ArrayObject $definitions) : \ArrayObject
```

Comment:
@Simperfit If the code structure does not lend itself well to the new requirements, don't try to shoehorn things into the existing structure. :smile: